app/plugins/Wallet/js/helpers.js

```diff
@@ -5,7 +5,7 @@ const IPC = require('ipc');
 // Library for arbitrary precision in numbers
 const BigNumber = require('bignumber.js');
 // Ensure precision
-BigNumber.config({ DECIMAL_PLACES: 24 });
+BigNumber.config({ DECIMAL_PLACES: 30 });
```

Comment:
This change keeps full precision out to the millions?

Seems odd that it's a parameter at all?